components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -235,12 +247,37 @@
   position: absolute;
   right: 0;
   z-index: 10000;
-  min-width: 140px;
-  max-width: 255px;
```

Comment:
The legend used to be more or less fixed size (auto-scaling within 140-255px limits). Now there is anew control which lets the legend be as wide as necessary to avoid splitting a name into two lines, which I think is more convenient on (very) wide displays. By default it is more or less the same size as before, just now it is dynamically controlled (css classes or styles are added in JS depending on user clicks on control buttons - or lack there of)